lua/cps.lua

```diff
@@ -0,0 +1,28 @@
+return function (net_conf, numa_table)
+
+	-- Init the CPS configuration structure.
+	local cps_conf = gatekeeper.c.get_cps_conf()
+	if cps_conf == nil then
+		error("Failed to allocate cps_conf")
+	end
+
+	-- Change these parameters to configure the CPS block.
+	local kni_kmod_path =
+		"/home/cody/gatekeeper/dependencies/dpdk/build/kmod/rte_kni.ko"
+
+	-- Setup the CPS functional block.
+	cps_conf.lcore_id = gatekeeper.alloc_an_lcore(numa_table)
+
+	local ret = gatekeeper.c.lua_init_kni(kni_kmod_path,
+		net_conf.back_iface_enabled and 2 or 1)
```

Comment:
You're becoming Lua savvy! :)

BUT, shouldn't `run_cps()` be responsible for initializing the KNI?